Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsExecutionStrategy.cs

```diff
@@ -47,6 +47,8 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
                 CsProjFileSearchPattern);
 
             var project = new Project(csProjFilePath);
+            this.SetupFixturePath = $"{project.DirectoryPath}\\{SetupFixtureFileName}.cs";
```

Comment:
Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.